packages/toast/src/Toast.types.ts

```diff
@@ -11,7 +13,8 @@ type Variant = (typeof Variant)[keyof typeof Variant];
 export { Variant };
 
 export interface ToastProps
-  extends Omit<React.ComponentPropsWithoutRef<'div'>, 'title'> {
+  extends Omit<React.ComponentPropsWithoutRef<'div'>, 'title'>,
+    DarkModeProps {
```

Comment:
we never had a darkMode type?

Reply:
We never did anything with it. It was passed in through `...rest`, but it was applied directly to the DOM, triggering runtime error logs